Responsiveness issue on demo page fixed

sraghuvanshi opened the pull request:
Fixes #6 

**Short description of what this resolves:**
This resolves the responsive issue on the demo page.

**Changes proposed in this pull request:**
1. Font-size reduced in mobile view.
2. Table defined using borders so that data is clearly readable.
3. Footer margin-top added.
4. "Links" table headings provided, because without it an empty row was present which was making the table look a little incomplete.

**Screenshot:**
![Screenshot from 2020-03-08 01-45-13](https://user-images.githubusercontent.com/48951339/76151760-88105780-60de-11ea-86b3-8cf9d0244225.png)

MatthewVita commented:
@bradymiller 

Ah, I can't wait to rid of those `&nbsp;` in favor a an actual css solution...

This change looks good to me. Feel free to merge.

Thanks,
-m

sraghuvanshi committed:
Table headings added

sraghuvanshi committed:
Front-end issues fixed

1. Font-size reduced from 1rem to 0.8rem in mobile view only.
2. Table defined using border
3. Footer margin-top provided

bradymiller commented:
hi @sraghuvanshi . Looks great on testing! Bringing this in now and it should up on the website in a couple minutes. Thanks! -brady

sraghuvanshi commented:
You're welcome sir :smiley:

Sir, I want to discuss a custom proposal idea. I was thinking it will be better if I discuss it with you first. Should I drop you an email? 
Thanks

bradymiller commented:
hi @sraghuvanshi , feel free to DM or email me (brady.g.miller@gmail.com).

sraghuvanshi commented:
Sir, I have sent an email. Hoping to hear from you.
Thanks a lot.
